src/main/java/gg/uhc/uhc/ShutdownListenerRegistry.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Project: UHC
+ * Class: gg.uhc.uhc.inventory.IconInventory
```

Comment:
Oh shit I just noticed what you meant with the class name.

Yeah I copied the headers as I didn't setup automatic header creation yet. Gonna fix that quickly